pkg/constants/constants.go

```diff
@@ -39,6 +39,9 @@ const (
 
 	// CheckpointName is the name of the object in each namespace in which the namespace's backup information is stored.
 	CheckpointName = "hive"
+
+	// AnnotationSyncsetPause is a annotation used by clusterDeployment, if it's true, then we will disable syncing to a specific cluster
+	AnnotationSyncsetPause = "hive.openshift.io/syncset-pause"
```

Comment:
s/AnnotationSyncsetPause/SyncsetPauseAnnotation/ . IMO `SyncsetPauseAnnotation` is a better name.